base/utils/DESCRIPTION

```diff
@@ -23,38 +23,36 @@ Description: The Predictive Ecosystem Carbon Analyzer
     investigation.
 Imports:
     abind (>= 1.4.5),
-    coda (>= 0.18),
-    data.table,
     dplyr,
-    ggplot2,
     lubridate (>= 1.6.0),
     magrittr,
     ncdf4 (>= 1.15),
     PEcAn.logger,
     PEcAn.remote,
-    PeriodicTable,
-    plyr (>= 1.8.4),
     purrr,
-    randtoolbox,
-    raster,
     RCurl,
-    rjags,
-    sp,
     stringi,
     udunits2 (>= 0.11),
-    xtable,
     XML
 Suggests:
+    coda (>= 0.18),
     MASS,
+    data.table,
+    ggplot2,
```

Comment:
Should be in alphabetical order

Reply:
Good catch. Addressed in 7a87488aff42f720bcf7deef05d54fe03c93f07f.